lib/reqTemplate.js

```diff
@@ -75,12 +71,32 @@ function prepareTemplateString(template, defaultLookupPath) {
  * @returns {Function} a template resolver for the provided template
  */
 function compileTemplate(templateSpec, setValue, reqPart) {
+    function createTAssemblyTemplate(template) {
+        if (/^(?:\$\.)?(?:[a-zA-Z][\w-]*)(?:\.[a-zA-Z][\w-]*)*$/.test(template)) {
+            // It's an ordinary template without default value
+            return [['raw', prepareTemplateString(template, reqPart)]];
+        }
+        var condTemplate = new RegExp('^((?:\\$\.)?(?:[a-zA-Z][\\w-]*)(?:\\.[a-zA-Z][\\w-]*)*)' +
```

Comment:
Idem. This one is even longer and with a lot of back-slashes. Not easy to read at all. Let's help fellow coders :)